sunkit_image/asda.py

```diff
@@ -11,410 +11,379 @@
 
 from sunkit_image.utils import calculate_gamma, points_in_poly, reform2d, remove_duplicate
 
-__all__ = ["Asda", "Lamb_Oseen"]
+__all__ = [
+    "gen_vel",
+    "gamma_values",
+    "center_edge",
+    "vortex_property",
+    "get_grid",
+    "get_vtheta",
+    "get_vradial",
+    "get_vxvy",
+]
+
+
+def gen_vel(vx, vy, i, j, r=3):
+    """
+    Given a point ``[i, j]``, generate a velocity field which contains a region
+    with a size of ``(2r+1) x (2r+1)`` centered at ``[i, j]`` from the original
+    velocity field ``self.vx`` and ``self.vy``.
+
+    Parameters
+    ----------
+    vx : `numpy.ndarray`
+        Velocity field in the x direction.
+    vy : `numpy.ndarray`
+        Velocity field in the y direction.
+    i : `int`
+        first dimension of the pixel position of a target point.
+    j : `int`
+        second dimension of the pixel position of a target point.
+    r : `int`, optional
+        Maximum distance of neighbor points from target point.
+        Default value is 3.
+
+    Returns
+    -------
+    `numpy.ndarray`
+        The first dimension is a velocity field which contains a
+        region with a size of ``(2r+1) x (2r+1)`` centered at ``[i, j]`` from
+        the original velocity field ``self.vx`` and ``self.vy``.
+        the second dimension is similar as the first dimension, but
+        with the mean velocity field subtracted from the original
+        velocity field.
+    """
+    if vx.shape != vy.shape:
+        raise ValueError("Shape of velocity field's vx and vy do not match")
+    if not isinstance(r, int):
+        raise ValueError("Keyword 'r' must be an integer")
+    vel = np.array(
+        [[vx[i + im, j + jm], vy[i + im, j + jm]] for im in np.arange(-r, r + 1) for jm in np.arange(-r, r + 1)],
+    )
+    return np.array([vel, vel - vel.mean(axis=0)])
 
 
-class Asda:
+def gamma_values(vx, vy, factor=1, r=3):

```

Comment:
```suggestion
def calculate_gamma(vx, vy, factor=1, r=3):
```

Reply:
couldn't rename it to calculate_gamma since there was already a function with that name, so I named it calculate_gamma_values